src/libDirectoryService/FinalBlockPostProcessing.cpp

```diff
@@ -281,6 +281,7 @@ void DirectoryService::ProcessFinalBlockConsensusWhenDone() {
   if (toSendPendingTxn) {
     m_mediator.m_node->SendPendingTxnToLookup();
   }
+  m_mediator.m_node->ClearUnconfirmedTxn();
```

Comment:
want to consider clearing `PendingAndDroppedTxn` as well

Reply:
Actually is you see,`Unconirmed txn` are only added in shard nodes. So, it is already empty in lookup and vice versa for `PendingAndDropped`